resolwe_bio/docker_images/base/Dockerfile.ubuntu-14.04

```diff
@@ -0,0 +1,65 @@
+FROM docker.io/resolwe/base:ubuntu-14.04
+
+MAINTAINER Genialis <dev-team@genialis.com>
+
+# Add common build helpers.
+ADD common.sh /var/cache/build/packages-manual/common.sh
```

Comment:
Ok, I see.

If you remove it at the end, then I'm fine with `/build`. Otherwise, I would prefer something like `/opt/genialis/docker-build-helpers`.